index.js

```diff
@@ -49,5 +49,59 @@ module.exports = {
         reason: e.toString()
       };
     }
+  },
+
+  getOptions: function() {
+    return [
+      {
+        name: 'Fake schema',
+        id: 'schemaFaker',
```

Comment:
is this used only for schema faking? using examples/defaults instead of schemas isn't technically faking.

Reply:
also, all the names should feel the same
some sound like instructions to the user - `Set root request body type`, some sound like instructions to the converter - `Collapse folder...`